pkg/syncthing/server.go

```diff
@@ -82,16 +82,21 @@ func (s *Server) Refresh() error {
 	return nil
 }
 
-// Update takes the current config, sets the server's config to that and
-// restarts the process so that it is live.
+// Update takes the current config, sets the server's config to that.
 func (s *Server) Update() error {
 	if _, err := s.client.NewRequest().
 		SetBody(s.Config).
 		Post("system/config"); err != nil {
 		return err
 	}
 
-	return s.Restart()
+	if _, err := s.client.NewRequest().
```

Comment:
I don't think you need this?

Reply:
If the path is different, I think I need a new client no? I guess I could just reuse the current one.

Reply:
I'll keep it for easy of reading for now I guess.